src/classes/frModel.cls

```diff
@@ -214,10 +213,14 @@ public with sharing abstract class frModel {
             }
             record.put(field, value);
         } catch (Exception ex) {
-            insert new Error__c(Error__c = 'Field mapping exception. Object type: '+ record.getSObjectType().getDescribe().getName()
+            try {
+                insert new Error__c(Error__c = 'Field mapping exception. Object type: '+ record.getSObjectType().getDescribe().getName()
                                 +' Record Id: '+record.Id+' - Funraise Id: '+ funraiseId + ' - Field: '+field.getDescribe().getName()+' - Value: '+value
                                 +' Exception: '+ex
                                );
+            } catch (Exception ignore) {
```

Comment:
Was there an exception being encountered on inserting the Error__c record?  I think that's concerning and should be addressed rather than catching an exception and ignoring it here, or just debugging it